### PR TITLE
[2.13.x] DDF-3945 Added permissions for deleting default users and to execute …

### PR DESCRIPTION
**Pull request by blen-desta**
Backport of https://github.com/codice/ddf/pull/3381

#### What does this PR do?
Added permissions for deleting default users and to executing jaas commands

#### Who is reviewing it? 
@bakejeyner  @brjeter 

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl 
@stustison 
@vinamartin

#### How should this be tested? (List steps with links to updated documentation)
1. Make sure the default users are deleted by the scheduler: ask me if you have questions on how to get that done
2. Make sure jaas commands work:
- Encrypting password
  - In the `etc/org.apache.karaf.jaas.cfg` file, change `encryption.enabled` to true
  - Make sure passwords in the `users.properties` file are encrypted
- Adding users:
  - `jaas:realm-manage --index 1`
  - `jaas:user-add foo bar`
  - `jaas:update`
  - `jaas:realm-manage --index 1`
  - `jaas:user-list` - see foo as a user
- Deleting users:
  - `jaas:realm-manage --index 1`
  - `jaas:user-delete foo`
  - `jaas:update`
  - `jaas:realm-manage --index 1`
  - `jaas:user-list` - foo is deleted as a user

#### What are the relevant tickets?
[DDF-3945](https://codice.atlassian.net/browse/DDF-3945)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 1]
```diff
@@ -256,6 +260,10 @@ grant codeBase "file:/admin-core-insecuredefaults/org.apache.felix.scr/org.apach
     permission java.io.FilePermission "${ddf.home.perm}etc${/}artemis.xml", "read";
 }
 
+grant codeBase "file:/admin-core-insecuredefaults" {
```
Comment (vinamartin) [inline distribution/ddf-common/src/main/resources/security/default.policy]: ❓ Is this just the one that deletes it (or rather blanks out the file)?

Comment (blen-desta) [reply distribution/ddf-common/src/main/resources/security/default.policy]: Blanks out the file

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
build now

**Comment by brjeter** [comment]
heroed successfully ✅

**Pull request merged**